core/integration/steps/transfers.go

```diff
@@ -210,7 +210,9 @@ func rowToRecurringTransfer(r RowWrapper) *types.RecurringTransfer {
 		teams := []string{}
 		ntop := ""
 		if entityScope == proto.EntityScope_ENTITY_SCOPE_TEAMS {
-			teams = strings.Split(r.MustStr("teams"), ",")
+			if r.HasColumn("teams") {
+				teams = strings.Split(r.MustStr("teams"), ",")
+			}
 			if len(teams) == 1 && teams[0] == "" {
 				mkts = []string{}
```

Comment:
only half related but I think this should be teams = []string{}

Reply:
This was already there, and `teams = []string{}` is set a couple of lines above. I'll check if `mkts` is being used (I imagine it is - still, I'll check)

Reply:
removed